Libraries/StyleSheet/processTransform.js

```diff
@@ -165,6 +165,11 @@ function _validateTransform(key, value, transformation) {
         stringifySafe(transformation),
       );
       break;
+    case 'perspective':
```

Comment:
it will be easy to reason out and maintain in future to have the number test in perspective rather than falling through default.